osquery/config/parsers/kafka_topics.h

```diff
@@ -0,0 +1,28 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include <osquery/config.h>
+
+namespace osquery {
+
+const std::string kKafkaTopicParserRootKey("kafka_topics");
```

Comment:
nit, move the definition into a cpp implementation

Reply:
Right, but can you extern it here? https://github.com/facebook/osquery/blob/master/include/osquery/database.h#L42

Then define it in the implementation file.